src/util.js

```diff
@@ -23,11 +23,37 @@ export const align = props => {
   return null
 }
 
+export const center = props => {
+  if (props.center || (props.centerX && props.centerY)) {
+    return {
+      top: '50%',
+      bottom: 'auto',
```

Comment:
This branch will apply only to the cases `<Absolute center />` and `<Absolute centerX centerY />`. 
For the `<Absolute centerX bottom />` case a `centerX` will set only the `left` and `right`. If we don't set the `left` and `right` together we can meet situation `<Absolute centerX right />` there not either `centerX` or `right` will work correctly.